### DISCUSSION TITLE
Input Sources of Task 1

### DISCUSSION THREAD

**Discussion by jiang-yiwei** [surgical-robotics-ai/surgical_robotics_challenge#54] [Q&A] [answered]
Hi,

I am cleaning up my code for the final submission and I'd like to confirm the inputs I used in my program are allowed and make sure the sources will be there during the evaluation.

1. Left and right image streams from `/ambf/env/cameras/cameraL/ImageData` and `/ambf/env/cameras/cameraR/ImageData`.
2. Return value from member function `get_T_c_w()` of object `ECM(c, "CameraFrame")`.

Thanks,
Yiwei

**Comment by adnanmunawar** [comment, marked as answer]
Hello Yiwei,

Thanks for the question, we are looking forward to receiving your submission. Yes, all these interfaces that you mentioned will be available for during the evaluation. Best of luck :)

**Reply by jiang-yiwei** [reply]
Thank you so much, Adnan!
